ansible/configs/sap-integration/files/pv-user.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
What is this being used for? I don't see these vars defined anywhere.

Reply:
done